src/wayland/structure.md

```diff
@@ -54,12 +54,12 @@ In the end, this kind of hierachy can be expected:
 
 *(The details of what these objects are and do will come later, in the [core protocol][] section.)*
 
-[core protocol]: ./wayland/p_core.html
+[core protocol]: ./p_core.html
 
 ## Protocol extensions
 
 The set of objects and the list of their requests and events is defined in
-[an XML file][wayland spec]. This approach allows wayland protocol extensions to be defined easily.
```

Comment:
My bad. In momentum of correcting I just inlined this link but it _already works_. That was the limitation I was talking about (the fact that rust doc might not be able to see that it is an external link to the document).